game/src/core/systems/LevelSystem.java

```diff
@@ -239,10 +240,12 @@ private Optional<ILevel> isOnDoor(final Entity entity) {
   }
 
   private void playSound() {
-    Sound doorSound = Gdx.audio.newSound(Gdx.files.internal(SOUND_EFFECT));
-    long soundId = doorSound.play();
-    doorSound.setLooping(soundId, false);
-    doorSound.setVolume(soundId, 0.3f);
+    if (DOOR_SOUND == null) {
```

Comment:
@tgrothe gibt es einen grund, warum du die funktionalität vom neuen soundplayer hier (mehr oder weniger) nochmal neu nachbaust, statt den soundplayer direkt zu nutzen?

Reply:
der grund war, dass man im `game` nicht auf `dungeon` zugreifen kann. wird bräuchten den soundplayer auch im `game`.

Reply:
naja, das rechtfertigt doch dann genau das, oder?

der bug steckt im ganzen projekt, verstreut über viele stellen. der fix sollte also so angelegt sein, dass alle diese stellen korrigiert werden. wenn das bedeutet, den neuen soundhandler nach `core` zu schieben, dann ist das eben so..

Reply:
danke, ist nun in: cc17bff1

Reply:
ich wollte die intrinsische ordnung nicht durcheinanderbringen... 🤣 (spaß)